modules/benchmark/benchmark_runner.py

```diff
@@ -140,7 +162,7 @@ def _create_configurations(self, num_scenarios=None):
                     benchmark_configs.append(benchmark_config)
         return benchmark_configs
 
-    def run(self, viewer=None, maintain_history=False):
+    def run(self, viewer=None, maintain_history=False, stage_dir=None):
```

Comment:
what does stage_dir imply?

Reply:
It defines the directory where intermediate results (stages) are stored.

Reply:
https://github.com/bark-simulator/bark/blob/7a21ff1ccf2e430ee3756c13708df2562d7d5593/modules/benchmark/benchmark_runner.py#L178-L188

Reply:
Okay, just talked to Klemens about that. Seems fine for me.